### DISCUSSION TITLE
Power consumption - power bank

### DISCUSSION THREAD

**Discussion by petrklic16** [jomjol/AI-on-the-edge-device#2290] [General]
Hi,

 i would like to ask you.
I have ESP32-CAM module to water meter and in water meter there isn't any power... i"m using power bank, but from this poweer bank, ESP works only about 2-3 days. Is there any chance how to reduce power ?

https://github.com/jomjol/AI-on-the-edge-device

Thank you

**Comment by caco3** [comment]
No, there is not much we could do.
The electronics are not made to be low-power.
